omod/src/main/java/org/openmrs/module/ugandaemr/web/controller/CarePanelController.java

```diff
@@ -0,0 +1,459 @@
+package org.openmrs.module.ugandaemr.web.controller;
```

Comment:
I dont  think this is a necessary endpoint given that [this PR  ](https://github.com/METS-Programme/esm-ugandaemr-core/pull/185) By @Daphne210 uses the fhir end point without use of this custom endpoint